apps/req.c

```diff
@@ -255,7 +255,7 @@ int req_main(int argc, char **argv)
     int informat = FORMAT_PEM, outformat = FORMAT_PEM, keyform = FORMAT_PEM;
     int modulus = 0, multirdn = 0, verify = 0, noout = 0, text = 0;
     int nodes = 0, newhdr = 0, subject = 0, pubkey = 0, precert = 0;
-    long newkey = -1;
+    long newkey_len = -1;
```

Comment:
This kind of mechanical rename is a little easier to verify if it's done in a dedicated commit.  No need to change this one, since it's already done, but maybe in the future if it comes up again.

Reply:
You are right, will do in the future.